src/translations/nbTranslations.json

```diff
@@ -1034,6 +1035,7 @@
     "new_registration": "Ny registrering",
     "public_page": {
       "about_registration": "Om registreringen",
+      "alternative_title": "Alternativ tittel",
```

Comment:
tenker vi kan gjenbruke teksten fra `description` her, da jeg ikke ser for meg at de skal ha ulike strenger en dag?
```suggestion
```